evm/contracts/examples/ConversionRate.sol

```diff
@@ -0,0 +1,111 @@
+pragma solidity 0.4.24;
+
+import "../ChainlinkClient.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract ConversionRate is ChainlinkClient, Ownable {
+  uint256 constant private ORACLE_PAYMENT = 1 * LINK; // solium-disable-line zeppelin/no-arithmetic-operations
+
+  struct Answer {
+    uint256 expectedResponses;
+    uint256[] responses;
+  }
+
+  uint256 public currentRate;
+  bytes32[] public jobIds;
+  address[] public oracles;
+
+  uint256 private answerCounter = 1;
+  uint256 private latestCompletedAnswer;
+  mapping(bytes32 => uint256) private requestAnswers;
+  mapping(uint256 => Answer) private answers;
+
+  constructor(address _link, address[] _oracles, bytes32[] _jobIds)
+    public
+    Ownable()
+  {
+    setChainlinkToken(_link);
+    updateOracles(_oracles, _jobIds);
+  }
+
+  function update()
```

Comment:
I really like this pattern for creating multiple requests, and I think I'm going to steal it for future examples. I just don't like the name `update()`. Would `createRequests()` be any better?